chart/openfaas/values.yaml

```diff
@@ -19,7 +19,7 @@ queueWorker:
 
 # images
 images:
-  controller: functions/faas-netesd:0.4.6
+  controller: functions/faas-netesd:0.4.7
```

Comment:
0.4.7 doesn't exist yet: https://hub.docker.com/r/functions/faas-netesd/tags/

Reply:
Yes this is problematic - I can't merge the change due to this. Could you revert this part and then after I release do a new PR to helm / YAML?

Reply:
Yep I'll submit this separately.